client/packages/common/src/ui/components/footer/ActionsFooter.tsx

```diff
@@ -41,6 +44,12 @@ export const ActionsFooter: FC<ActionsFooterProps> = ({
     );
   };
 
+  actions.push({
+    label: t('label.clear-selection'),
+    icon: <MinusCircleIcon />,
+    onClick: clearSelected,
+  });
```

Comment:
do you reckon it could be separate, right aligned?

Reply:
Looks awkward to me to have it right-aligned... but sure XD